### DISCUSSION TITLE
24.05.13 Progress Meeting

### DISCUSSION THREAD

**Discussion by ssutjipto** [Woolfrey/software_robot_library#112] [Meetings]
#### Chair:
@ssutjipto 

#### Attendees:
@Woolfrey 
@ssutjipto 
_____

## Announcements
Nothing to announce.

## Agenda
_Note: Last week's actions become this week's agenda_

| Item | Responsible | Issue | Outcome |
|-------|------------------|----------|--------------|
| Resolve server base class structure | @Woolfrey | [Link](https://github.com/Woolfrey/server_serial_link/issues/1) | Solution found for issue, now uses template class |
| Check indenting, resolve issue when complete | @ssutjipto | #103 | Will do after this meeting |
| Begin work on dynamic controller | @ssutjipto |  | In progress |

## Additional Discussion
| Topic | Raised By | Issue | Outcome |
|-------|------------------|----------|--------------|
| What is the physical interpretation of the weighting in the non-redundant Cartesian control optimisation | @Woolfrey @ssutjipto  |  | Still requires further discussion, issue will be created and linked here |
| Namespace for classes | @Woolfrey |  | Issue will be created and linked here |

## Actions
| Action | Responsible | Tag |
|----------|------------------|-------|
| Create class member for joint damping vector | @ssutjipto | |
| Continue ActionServer base class | @Woolfrey | [Link](https://github.com/Woolfrey/server_serial_link/issues/1) |

**Comment by Woolfrey** [comment]
Add this to the code for the redundant dynamic control:
```
Bruyninckx, H., & Khatib, O. (2000, April). Gauss' principle and the dynamics of redundant and constrained manipulators. In Proceedings 2000 ICRA. Millennium Conference. IEEE International Conference on Robotics and Automation. Symposia Proceedings (Cat. No. 00CH37065) (Vol. 3, pp. 2563-2568). IEEE.
```
